app/javascript/react/components/BrandShowContainer.js

```diff
@@ -29,19 +32,44 @@ const BrandShowContainer = (props) => {
       getPizzas()
   }, [])
 
+
   const pizzaTiles = brand.pizzas.map( (pizza) => {
     return(
       <BrandShowTile name={pizza.product_name} id={pizza.id} key={pizza.id} />
     )
   })
-  
+
+  const addPizza = async (formPayload) => {
+    const response = await fetch("/api/v1/pizzas/", {
+      credentials: "same-origin",
+      method: "POST",
+      headers: {
+        "Accept": "application/json",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify(formPayload),
+    })
+    const parsedNewPizza = await response.json()
+    if(parsedNewPizza.errors[0] === 'Pizza added succesfully.') {
```

Comment:
if we wanted to check that the POST was successful we can check if `parsedNewPizza.pizzas`, then set state

Reply:
Yeah I dont know why we always assumed we had to make successful message!